app.js

```diff
@@ -1,22 +1,81 @@
 const fs = require('fs');
+const path = require('path');
 const acorn = require('acorn');
 const escodegen = require('escodegen');
 const argv = require('boring')();
+const cp = require('child_process');
+const { stripIndent } = require('common-tags');
 
 // This is a random identifier not used anywhere else
 const blankLineMarker = "// X0k7FEu5a6!bC6mV";
 
-const moduleNames = argv._;
+if (argv._[0] === 'undo') {
+  cp.execSync('git reset --hard && git clean -df');
+  process.exit(0);
+}
+
+try {
+  cp.execSync('git status', { encoding: 'utf8' });
+} catch (e) {
+  fail(stripIndent`
+    This project does not appear to be using git. For your protection and to simplify
+    certain operations this tool can only be used in a git repository.
+  `);
+}
 
-for (const module of moduleNames) {
-  processModule(module);
+if (isSingleSiteProject()) {
+  // Only touch directories, don't mess about with regular files in lib/modules (WTF)
+  // or symlinks in lib/modules (double WTF, but outside our remit)
+  let moduleNames = fs.readdirSync('lib/modules').filter(moduleName => fs.lstatSync(`lib/modules/${moduleName}`).isDirectory());
+  moduleNames.forEach(name => {
+    rename(`lib/modules/${name}`, `modules/${name}`);
+  });
+  try {
+    fs.rmdirSync('lib/modules');
+  } catch (e) {
+    if (e.code === 'ENOTEMPTY') {
+      console.error(stripIndent`
+        "lib/modules" is not empty after moving modules to "modules". You probably
```

Comment:
Not a fatal error. Needed to clarify why lib/modules hangs around if you have extraneous guff like .js files in it.